### ISSUE TITLE
Potential Data Leakage Concern in Training Process

### ISSUE THREAD

**Issue by hutchinsonian** [PRIME-RL/SimpleVLA-RL#34]
Hi,

I'm looking into your project and it's been very insightful. I have a question regarding the data loading process for the RL training and wanted to bring a potential issue to your attention.

In the file `verl/workers/rollout/rob_rollout.py`, specifically at [this line](https://github.com/PRIME-RL/SimpleVLA-RL/blob/dd648c9baafb9568033c6237f954b5b8273d001d/verl/workers/rollout/rob_rollout.py#L128), the initial states for the environment are loaded using the following code:

```python
benchmark_dict = benchmark.get_benchmark_dict()
task_suite = benchmark_dict[task_name]()
task = task_suite.get_task(task_id)
initial_states = task_suite.get_task_init_states(task_id)
initial_state = initial_states[trial_id]
```

My concern is that `task_suite.get_task_init_states(task_id)` appears to load the initial states from the LIBERO benchmark's test set. If this is the case, it would mean that the RL training is being performed on the test data, which could lead to a data leakage issue where the model is trained on the same data it is evaluated on.

For training, my understanding is that the initial states should correspond to the training data. This data is available at [yifengzhu-hf/LIBERO-datasets on Hugging Face](https://huggingface.co/datasets/yifengzhu-hf/LIBERO-datasets). The intended process would be to load the demos from this dataset and then set the environment's initial state accordingly, for example, by using a method like `env.set_init_state(orig_states[0])`.

Could you please clarify if my understanding of the code is correct? If so, this is a potential issue that could affect the validity of the training and evaluation results.

Thank you for your great work on this project and for your time.
@zhan72
